media/css/firefox/family-nav.less

```diff
@@ -785,4 +787,28 @@ html[lang^='en'] .fxfamilynav .subsubnav a {
             display: block;
         }
     }
+
+    // remove ability to hover and show subnav
+    // must be done in JS to avoid hover/tab focus text overlap
+    // V1 only - V2 has no top level links to focus
+    #fxfamilynav {
+        .primarynav {
+            li {
+                &:hover,
+                &:focus {
+                    .subnav {
+                        display: none;
+                    }
+                }
+            }
+
+            .primary-link:focus + .subnav {
+                display: none;
+            }
+        }
+
+        .subnav.active {
+            display: block !important;
```

Comment:
It unfortunately is, as it needs to override the `display: none;` values directly above, which in turn are overriding the no JS `display` values set prior.